client/src/partials/price_list/modal.html

```diff
@@ -13,36 +13,53 @@
       </div>
     </div>
 
-    <div class="form-group" ng-class="{ 'has-error' : ModalForm.$submitted && ModalForm.value.$invalid }">
+    <div class="form-group" ng-class="{ 'has-error' : ModalForm.$submitted && (ModalForm.value.$invalid || ModalCtrl.error) }">
       <label class="control-label">{{ 'FORM.LABELS.VALUE' | translate }}</label>
       <input type="number" class="form-control" name="value" ng-model="ModalCtrl.data.value" required>
       <div class="help-block" ng-messages="ModalForm.value.$error" ng-show="ModalForm.$submitted">
         <div ng-messages-include="partials/templates/messages.tmpl.html"></div>
       </div>
     </div>
 
-    <div class="form-group" ng-class="{ 'has-error' : ModalForm.$submitted && ModalForm.is_percentage.$invalid }">
-      <label class="control-label">
-        <input type="checkbox"  name="is_percentage" id="is_percentage" ng-true-value="1" ng-false-value="0" ng-model="ModalCtrl.data.is_percentage">  {{ 'FORM.LABELS.IS_PERCENT' | translate }}
+    <div class="checkbox">
+      <label>
+        <input type="checkbox" name="is_percentage" id="is_percentage" ng-true-value="1" ng-false-value="0" ng-model="ModalCtrl.data.is_percentage">  {{ 'FORM.LABELS.IS_PERCENT' | translate }}
       </label>
     </div>
 
     <div class="form-group" ng-class="{ 'has-error' : ModalForm.$submitted && ModalForm.inventory_uuid.$invalid }">
       <label class="control-label">{{ "FORM.LABELS.INVENTORY" | translate }}</label>
-      <select class="form-control" name="inventory_uuid" ng-model="ModalCtrl.data.inventory_uuid" id="inventory_uuid" ng-options="i.uuid as i.label for i in ModalCtrl.inventories | orderBy:'label'" required>
-        <option value="" disabled> -- {{ 'FORM.SELECT.INVENTORY' | translate }} -- </option>
-      </select>
+      <ui-select
+        name="inventory_uuid"
+        ng-model="ModalCtrl.data.inventory_uuid"
+        ng-model-options="{ 'debounce' : 200  }"
```

Comment:
This should be removed or addressed here https://github.com/IMA-WorldHealth/bhima-2.X/issues/972.